sqltoolsservice.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
-VisualStudioVersion = 15.0.26510.0
+VisualStudioVersion = 15.0.26730.10
```

Comment:
This changed automatically since I'm using the latest version of Visual Studio 2017. As far as I know, there shouldn't be any negative effects from this since the MinimumVisualStudioVersion is still the same